src/GraphExplorer.js

```diff
@@ -0,0 +1,313 @@
+/**
+ * Created by valeriy.abornyev on 11/9/2015.
+ */
+
+var vis = require('vis'),
+    $ = require('jquery'),
+    ElementsCollection = require('./ElementsCollection');
+
+
+    function GraphExplorer(settings) {
+        var visData;
+        var status = false;
+        var statusSuggestion = false;
+        var self = this;
+        this.container = settings.container;
+        this.options = settings.options;
+        this.inputData = {};
+
+
+        $.ajax({
+            method: 'get',
+            url: settings.data,
```

Comment:
Let's make it's clean

In our readme we have a description of data object which we pass to GraphExplorer 

1. If you do something different, please update README.md first.
1. let's  name things properly , if it's url let it be url but not a data
1. I don't remember that we agreed on  passing URL as part of our API, but it's ok to support both options